### ISSUE TITLE
Missing parent role="tablist", ce_accordionStart.html5

### ISSUE THREAD

**Issue by cmichael-de** [contao/contao#1703]
In Contao 4.4.47 the Webdeveloper audit shows an accessibility error caused by a missing parent role="tablist"  in ce_accordionStart.html5. I fixed it for me. I just created a new ce_accordionStart.html5 template with the missing role:

`<section role="tablist" class="<?= $this->class ?> ce_accordion block"<?= $this->cssID ?><?php if ($this->style): ?> style="<?= $this->style ?>"<?php endif; ?>>`

Is that already implemented in higher Contao Versions after 4.4.47? 

Issue related Link: https://www.w3.org/TR/wai-aria-1.1/#tab

**Comment by MacKP** [comment]
Ah, nice catch. In one way @cmichael-de is correct. But the real problem is here: this should be not an tablist. Accordions are no tabs ;-) 
Here is an quick example for an accessible accordion: https://getbootstrap.com/docs/4.4/components/collapse/#accordion-example (a button in a headline). Without JS it should be the headline and text.
Contao uses an outdatet version of this... but the modern way is not backwards compatible here.
Sadly i have no clue what we should do here now, because it is a bug, but the fix wouldn't be the best way to fix it.
BTW: The fix should be set the role in the JS not in the template (for users without JS, there should be not role here).
Are there other opinions/options here @NinaG @kikmedia ?

**Comment by leofeyer** [comment]
As discussed in Mumble on August 27th: 

> But the real problem is here: this should be not an tablist. Accordions are no tabs ;-)

That is correct. We could remove the roles in the `moo_accordion` template, however, the jQuery script adds the roles dynamically, so we either need to use a more current version or another JS library – preferably not depending on jQuery or MooTools (e.g. https://github.com/oncode/handorgel).

**Comment by richardhj** [comment]
Adding `aria-controls=""` to the toggler and `aria-expanded="" aria-labelledby=""` to the div should be sufficient, no?
We may add `role=button` to the toggler.

**Comment by stefan-at-work** [comment]
Take a look at the solution from Matthias Ott: https://github.com/matthiasott/a11y-accordion-tabs

**Comment by Aybee** [comment]
Also ich würde hier erstmal das `role="tablist"` hinzufügen. Das ist beim originalen jQuery Accordion auch mit dabei. Bei uns wird es nicht aufgesetzt, weil das Accordion mit 
`$(document).accordion({...`
anstatt mit
`$('.ce_accordion').accordion({...`
initialisiert wird.

Mein Vorschlag (habe die fehlenden Klassen auch noch hinzugefügt):
```js
/* j_accordion.html5 */
...
create: function(event, ui) {
  ...
  $($(document).accordion('option', 'header')).parent().attr('role', 'tablist').addClass('ui-accordion ui-widget ui-helper-reset');
},
...
```

Hintergrund: Ich hatte gerade nen Kunden, der hatte bei der Website weiten Barrierearmutanalyse über 3000 Fehler diesbezüglich.
